deploy/olm-catalog/argocd-operator/0.3.0/argoproj.io_appprojects.yaml

```diff
@@ -1,7 +1,6 @@
 apiVersion: apiextensions.k8s.io/v1
```

Comment:
The changes to this file should be reverted. They are added by the make bundle step (not sure why, but they are in the 0.1 and 0.2 releases as well), which will be run as part of the release process.